src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -267,6 +268,15 @@ private function registerCommonConfiguration(ContainerBuilder $container, array
             $loader->load('symfony/uid.php');
         }
 
+        // symfony/serializer:7.3 added the NumberNormalizer
+        // symfony/framework-bundle:7.3 added the serializer.normalizer.number` service
+        // if symfony/serializer >= 7.3 and symfony/framework-bundle < 7.3, the service is registred
+        if (class_exists(NumberNormalizer::class) && !$container->has('serializer.normalizer.number')) {
```

Comment:
I suggest to just bump the minimum required versions to avoid supporting this edge case.

Reply:
we just bumped everything to 7.4 if I'm not mistaken